subprojects/test-runner/service/src/main/kotlin/com/avito/runner/reservation/DeviceReservationWatcherImpl.kt

```diff
@@ -12,11 +12,22 @@ internal class DeviceReservationWatcherImpl(
 ) : DeviceReservationWatcher {
 
     override suspend fun watch(deviceSignals: ReceiveChannel<Device.Signal>) {
+        val deployments: MutableMap<String, String> = mutableMapOf()
```

Comment:
Looks like we don't have to store this state there. We could store it directly in Claimer